firstuseauthenticator/templates/reset.html

```diff
@@ -0,0 +1,38 @@
+{% extends "page.html" %}
+
+{% block main %}
+
+<div class="container">
+<form action="{{post_url}}" method="post" role="form">
+  <h2 class="auth-form-header">
+    Change Password
+  </h2>
+  <div class='auth-form-body'>
```

Comment:
Since we aren't explicitly styling these, these class attributes can probably be removed.